examples/counter/android/src/main/res/drawable/ic_arrow_downward_white_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```

Comment:
This should be formatted so everything is on a separate line.

Reply:
done.  was just the way AS imports vector drawables